src/containers/Historical/Historical.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import './Historical.css'
+// import { connect } from 'react-redux'
+// import * as actions from '../../actions'
```

Comment:
Commented out due to Travis-CI treating warnings about unused variables as errors.